sympy/combinatorics/fp_groups.py

```diff
@@ -581,12 +591,18 @@ def _subnormal_series(G):
     _next_order = _next.order()
     while not igcd(_next_order, totient(_next_order)) == 1:
         # Loop until a square free ordered group is obtained.
-        elem = _next.generators[0]
-        _next = _next.subgroup(elem)
+        _parent = _next
+        elem = _parent.generators[0]
+        _next = _parent.subgroup(elem)
+        # Injective homomorphism between `_next` which is defined
+        # on another free_group and the parent group.
+        _homomorphism = homomorphism(_next, _parent, _next.generators[0], _parent.generaotrs[0], check=False)
```

Comment:
typo: `generators`. But beside that, I don't think this defined the injective homomorphism we want, in general.

Reply:
Since we are computing the presentation using one element of the generators, will that be changed? I meant, for example, if `x, y` are the generators of `G` and we compute the presentation of subgroup generated by `x`, will `x1` not correspond to `x` here? Or is it still gonna change?

Reply:
Oh right, it's a cyclic group and we know what it's generated by. In that case this should be fine.